crmd/lrm.c

```diff
@@ -1806,9 +1855,36 @@ construct_op(lrm_state_t * lrm_state, xmlNode * rsc_op, const char *rsc_id, cons
     op->timeout = crm_parse_int(op_timeout, "0");
     op->start_delay = crm_parse_int(op_delay, "0");
 
+    rsc = lrm_state_get_rsc_info(lrm_state, rsc_id, 0);
+
+    if (safe_str_eq(op->op_type, CRMD_ACTION_STATUS) && op->interval == 0 && rsc) {
+        lrm_state_update_rsc_metadata(lrm_state, rsc, NULL);
```

Comment:
I think it's unnecessary to notify about metadata action but we must call do_update_resource() in order to record operation result in cib. Otherwise tengine will wait for the timeout and will eventually raise 'Timed out' error.